File: Maths_And_Stats/series/sumofNnaturalnums.py
#  Sum of n natural Number in python3 ..

try:
    n=int(input("Enter the Nth term upto which you want to calculate the sum: "))
    a=1
    d=1
    ans = (n/2)*(2*a + (n-1)*d)
    print(ans)

except ValueError:
    print("Please enter a valid integer with base 10.")

File: Maths/Series/Sum_of_N/sumofNnaturalnums.cs
using System;  
public class Exercise3  
{  
    public static void Main() 
{
   int i,n,sum=0;
   Console.Write("\n\n");
   Console.Write("Input Value of terms : ");
   n= Convert.ToInt32(Console.ReadLine());   
   for(i=1;i<=n;i++)
   {
     sum+=i;
   }
   Console.Write("\nThe Sum of Natural Number upto {0} terms : {1} \n",n,sum);
  }
}